build/download-manifests/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/projectsyn/steward/pkg/images"
+)
+
+var crds = []string{
```

Comment:
Could probably be `const`, not sure what the convention is re `var`/`const` in Go.

```suggestion
const crds = []string{
```